environment.yml

```diff
@@ -4,19 +4,23 @@ channels:
 - conda-forge
 dependencies:
 - python<3.7
-- pip==19
+- pip=19
 - mkl
 - numpy
 - scipy
+- scikit-learn
```

Comment:
why scikit-learn?!

Reply:
because circleci told me that I need it to make nilearn run